File: Java/Activities/programs/RPSGame/RPS.java
package com.ibm.programs.RPSGame;

import java.util.Random;
import java.util.Scanner;

public class RPS {
    public static void main(String[] args) {
        Scanner Mee=new Scanner(System.in);
        Random Rand=new Random();
        String[] choices = {"Rock", "Paper", "Scissors"};
        try{
        System.out.println("Enter (0-Rock),(1-Papre),(2-Scissors)");
        int userchoice=Mee.nextInt();
        if(userchoice<0||userchoice>2){
            System.out.println("invalid choice Entered");
            return;
        }
        int computerChoice = Rand.nextInt(3);
        System.out.println("Enter the computer choice"+computerChoice);
        if(userchoice==computerChoice){
            System.out.println("Match draw!!!!");

        }else if((userchoice==0 && computerChoice==2)||
        (userchoice==1 && computerChoice==0)||
        (userchoice==2 && computerChoice==1)){
            System.out.println("You win!!!!");
        }else{
            System.out.println("Computer Wins!!!!");
        }
        }catch(Exception e){
            System.out.println("Invalid input enter valid inputs");
        }
    }
}

File: Python/Activities/python/RPS.py
import random

while True:

    user = input("Enter rock, paper, or scissors: ").lower()

    choices = ["rock", "paper", "scissors"]
    computer = random.choice(choices)

    print("Computer chose:", computer)

    if user == computer:
        print("It's a tie!")

    elif (user == "rock" and computer == "scissors") or \
         (user == "paper" and computer == "rock") or \
         (user == "scissors" and computer == "paper"):
        print("You win!")

    else:
        print("Computer wins!")

    play_again = input("Do you want to play again? (Yes/No): ").lower()

    if play_again != "yes":
        print("Game ended")
        break